python/cudf/cudf/core/dataframe.py

```diff
@@ -2830,6 +2830,22 @@ def replace(self, to_replace, replacement):
         -------
         result : DataFrame
             DataFrame after replacement.
+
+        Examples
+        --------
+        >>> import cudf
+        >>> gdf = cudf.DataFrame()
+        >>> gdf['id']= [0, 1, 2, -1, 4, -1, 6]
+        >>> gdf['id']= gdf['id'].replace(-1, None)
+        >>> gdf
+             id
+        0     0
+        1     1
+        2     2
+        3  null
+        4     4
+        5  null
+        6     6
```

Comment:
Would be nice to add some additional examples:

* Using `replace` to replace a value with a non-null value
* Using `replace` with list inputs for `to_replace` and `replacement` (allows replacing multiple values)
* Using `replace` with `dict` input for `to_replace` and `replacement` (allows specifying different replacements for different columns)

Reply:
Discussed offline and these will be added in a later PR. Marking resolved.